helm/node-health-chart/values.yaml

```diff
@@ -0,0 +1,6 @@
+image:
+  registry: quay.io
+  repository: giantswarm/node-health
+  tag: 0.1.0
```

Comment:
This is actually not going to work, since architect pushes our docker image to `[SHA]` tags. We only had external components before, so there were fixed tags. Maybe if you put `[SHA]` for the tag, architect will template this beforehand. Probably we should come up with something better here. @fgimenez WDYT?

Reply:
I really don't understand this here. How should it be? How do I get my stupid daemonset deployed?

Reply:
> Maybe if you put [SHA] for the tag, architect will template this beforehand. Probably we should come up with something better here. @fgimenez WDYT?

:+1: using `[[ .SHA ]]` for the image tag sounds good to me, architect will insert the right value in the template, and using SHAs for image tags is the right thing to do IMO.

Reply:
So `appVersion`, or should this be the Git SHA as well?

Reply:
Set the image tag to Git SHA.

Reply:
In this case `appVersion` doesn't make sense, there's no upstream app, it is a component of our own. For the image tag IMO it's better to use `[[ .SHA ]]`, our current tooling adds by default the a sha tag to the pushed images, so we can use it here.

The `version` field is only chart related, currently chart-operator doesn't expect any sha info in it, only a semver, hope this helps.